fix GEM geometry - HCAL emap - HCAL resoCorr HEP17 - SiStrip bad channels [91x version]

franzoni opened the pull request:
This PR will lead to the same set of GT's in 91X as this other PR https://github.com/cms-sw/cmssw/pull/17886 in do in 90x

Since in 91x, a set of updates
("Summer16 JECs, GEM reco geometry and ESEE Intercalib" - see PR https://github.com/cms-sw/cmssw/pull/17702 ; the equivalent 90x https://github.com/cms-sw/cmssw/pull/17649 is yet to be merged )
has already been merged,
the changes expected in this PR are fewer than have been observed for the 90x https://github.com/cms-sw/cmssw/pull/17886

The changes expected are described in the [further down in this chat](https://github.com/cms-sw/cmssw/pull/17905#issuecomment-286274828), since commits have been added

Commit:
Merged refs/pull/17649/head from repository cms-sw

Commit:
fix GEM geometry - HCAL emap - HCAL resoCorr HEP17 - SiStrip bad channels

Commit:
Merged refs/pull/17886/head from repository cms-sw

franzoni commented:
pelase test

cmsbuild commented:
A new Pull Request was created by  @franzoni (Giovanni Franzoni) for master.

It involves the following packages:

Configuration/AlCa


@ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please review it and eventually sign? Thanks.
@makortel, @Martin-Grunewald, @ghellwig, @tocheng this is something you requested to watch as well.
@muzaffar, @davidlange6, @smuzaffar you are the release manager for this.

cms-bot commands are listed here cms-sw/cmssw#13028

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

arunhep committed:
data GTs updated with fixed GEM reco tag

cmsbuild commented:
+1
Tested at: 015b97da259e3367872a9f5d9c483220301bb94f
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18365/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/015b97da259e3367872a9f5d9c483220301bb94f
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18365/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18365/git-merge-result

cmsbuild commented:
Comparison job queued.

arunhep committed:
MC Realistic GTs updated

arunhep committed:
Cosmic GT for 2018 updated

arunhep committed:
Cosmic GT updated

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18365/summary.html

cmsbuild commented:
Pull request #17905 was updated. @ghellwig, @arunhep, @cerminar, @cmsbuild, @franzoni, @mmusich, @davidlange6 can you please check and sign again.

franzoni commented:
please test

cmsbuild commented:
The tests are being triggered in jenkins.

franzoni commented:
This PR will lead to the same set of GT's in 91X as this other PR https://github.com/cms-sw/cmssw/pull/17886 in do in 90x

Since in 91x, a set of updates
("Summer16 JECs, GEM reco geometry and ESEE Intercalib" - see PR https://github.com/cms-sw/cmssw/pull/17702 ; the equivalent 90x https://github.com/cms-sw/cmssw/pull/17649 is yet to be merged )
has already been merged,
the changes expected in this PR are fewer than have been observed for the 90x https://github.com/cms-sw/cmssw/pull/17886

The changes expected are described in the following

# Summary of changes in Global Tags

## Data

   * **runI and runII data** : [90X_dataRun2_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v6) as [90X_dataRun2_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v6/90X_dataRun2_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data relval** : [90X_dataRun2_relval_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_relval_v6) as [90X_dataRun2_PromptLike_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_PromptLike_v6/90X_dataRun2_PromptLike_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII data prompt-like** : [90X_dataRun2_PromptLike_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_PromptLike_v6) as [90X_dataRun2_v5](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_v5) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_v6/90X_dataRun2_v5):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT** : [90X_dataRun2_HLT_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v4) as [90X_dataRun2_HLT_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_frozen_v4/90X_dataRun2_HLT_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runI and runII HLT relval** : [90X_dataRun2_HLT_relval_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v4) as [90X_dataRun2_HLT_relval_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLT_relval_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLT_relval_v4/90X_dataRun2_HLT_relval_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note


   * **runII HLT HI** : [90X_dataRun2_HLTHI_frozen_v4](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v4) as [90X_dataRun2_HLTHI_frozen_v3](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_dataRun2_HLTHI_frozen_v3) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_dataRun2_HLTHI_frozen_v4/90X_dataRun2_HLTHI_frozen_v3):
      * the GEM reco geometry is only available for 2017 runs (>= 287446); please @ianna take note



## Upgrade

   * **PhaseII realistic scenario** : [90X_upgrade2023_realistic_v8](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v8) as [90X_upgrade2023_realistic_v6](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2023_realistic_v6) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2023_realistic_v8/90X_upgrade2023_realistic_v6):
      * removed unnecessary geometry records
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/

   * **PhaseI 2017 cosmics peak scenario** : [90X_upgrade2017cosmics_realistic_peak_v17](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v17) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v17/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 cosmics scenario** : [90X_upgrade2017cosmics_realistic_deco_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v16) as [90X_upgrade2017cosmics_realistic_peak_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017cosmics_realistic_peak_v15) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017cosmics_realistic_deco_v16/90X_upgrade2017cosmics_realistic_peak_v15):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2017 design scenario** : [90X_upgrade2017_design_IdealBS_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v18) as [90X_upgrade2017_design_IdealBS_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_design_IdealBS_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_design_IdealBS_v18/90X_upgrade2017_design_IdealBS_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html


   * **PhaseI 2017 realistic scenario** : [90X_upgrade2017_realistic_v18](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v18) as [90X_upgrade2017_realistic_v16](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2017_realistic_v16) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2017_realistic_v18/90X_upgrade2017_realistic_v16):
      * updated geometry (fix GEM passive volume) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2841.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * HcalRespCorrs 1 -> 1.05 https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2843.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html


   * **PhaseI 2018 cosmics scenario** : [90X_upgrade2018cosmics_realistic_deco_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v15) as [90X_upgrade2018cosmics_realistic_peak_v12](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018cosmics_realistic_peak_v12) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018cosmics_realistic_deco_v15/90X_upgrade2018cosmics_realistic_peak_v12):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * sistrip noise switched from peak to DECO (the new default)
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

   * **PhaseI 2018 design scenario** : [90X_upgrade2018_design_IdealBS_v14](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v14) as [90X_upgrade2018_design_IdealBS_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_design_IdealBS_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_design_IdealBS_v14/90X_upgrade2018_design_IdealBS_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html


   * **PhaseI 2018 realistic scenario** : [90X_upgrade2018_realistic_v15](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v15) as [90X_upgrade2018_realistic_v13](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/gts/90X_upgrade2018_realistic_v13) with the following [changes](https://cms-conddb.cern.ch/cmsDbBrowser/diff/Prod/gts/90X_upgrade2018_realistic_v15/90X_upgrade2018_realistic_v13):
      * updated geometry (fix GEM passive volume and HCAL changes from code) https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2842.html
      * HcalElectronicsMap https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2838.html
      * sistrip bad components https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2837.html
      * beamstpot consistenet w/ alignement https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2852.html
      * ECAL: 2017 predictions for channel status, APD/Pn and alphas, pedestal, Trigger Primitives, selective readout, ES calibration - https://indico.cern.ch/event/615692/contributions/2484421/
      * Drift tubest alignment and alignmenet errors https://hypernews.cern.ch/HyperNews/CMS/get/calibrations/2851.html

cmsbuild commented:
+1
Tested at: ece771b4a9251dbf2708d0ceb0630bb227619e4a
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18381/summary.html

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/015b97da259e3367872a9f5d9c483220301bb94f
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18381/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18381/git-merge-result

cmsbuild commented:
Comparison job queued.

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-17905/18381/summary.html

franzoni commented:
**phaseI MC 2017:**
   * the change in geometry is expected to scatter differences widely, due to its impact on the pseudorandom number sequences
   * low level Sistrip plots hightly segmented -> can't see effect of the channel status changes over the 10 available events (plus the previous bullet)
   * HCAL vs eta hit occupancies are the same or compatible (the emap change is expected to have no effect on digi/rehits)
   * change in occupancy clear in EB, as a result of SelRead and Transparency/Pedestal changes

**phaseI MC 2018:**
   * same picture as 2017

**phaseII MC:**
   * the change in channel status is not picked up by the Phase2OuterTracker plots . Tracking is also totally unaffected --> expected

**data:**
   * 2012B and 2016B unaffected by replacing for GEM reco geometry the IOV 1-inf with 287446-inf

franzoni commented:
+1

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next master IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @muzaffar, @davidlange6, @smuzaffar
